src-python/lab-service/lab/service/lockkeeper.py

```diff
@@ -243,6 +243,24 @@ def nat_cleanup():
     return jsonify({'status': "iptables -t nat -F INPUT"})
 
 
+@app.route('/link/tc', methods=['POST'])
+def set_link_delay():
+    body = request.get_json()
+    cmd = 'tc qdisc add dev {} root netem delay {}ms'.format(body['interfaceName'], body['delayMs'])
+    container_name = 'lab_service-{}'.format(body['labId'])
+    execute_commands_in_container([cmd], container_name)
+    return jsonify({'status': 'ok'})
+
+
+@app.route('/link/tc/cleanup', methods=['POST'])
+def cleanup_link_delay():
+    body = request.get_json()
+    cmd = 'tc qdisc delete dev {} root'.format(body['interfaceName'])
```

Comment:
I don't like `interface name` in API. Client of lockkeeper (test code) should build correct interface name for lab service. Encapsulation is broken here.